apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -134,3 +223,20 @@ export const convertStudentServerData = (studentData, loginType, sectionId) => {
 export const convertStudentDataToArray = (studentData) => {
   return Object.values(studentData);
 };
+
+// Make a post request to edit a student.
+const updateStudentOnServer = (updatedStudentInfo, onComplete) => {
+  $.ajax({
+    url: `/v2/students/${updatedStudentInfo.id}/update`,
+    method: 'POST',
+    contentType: 'application/json;charset=UTF-8',
+    data: JSON.stringify(updatedStudentInfo),
+  }).done((data) => {
+    onComplete(data);
+  }).fail((jqXhr, status) => {
+    // We may want to handle this more cleanly in the future, but for now this
+    // matches the experience we got in angular
+    alert("Unable to save student");
+    console.error(status);
```

Comment:
Your failure case should also call the callback, otherwise it's easy to write code that hangs in a failure case.  I'd recommend adopting a node-style callback approach - `function(err, result)` where in a success case you get `(null, result)` and in an error case you get `(Error, null)`.